ooni/nettest.py

```diff
@@ -454,7 +446,11 @@ def checkAllTasksDone(self):
                   (self.doneTasks, self.tasks))
         if self.completedScheduling and \
                 self.doneTasks == self.tasks:
-            self.allTasksDone.callback(self.doneTasks)
+            if self.allTasksDone.called:
+                log.err("allTasksDone was already called. This is probably a bug.")
```

Comment:
Yeah a stacktrace here doesn't really provide much insight into the problem and it was the case in the past that we would throw it (in the form of an already called error).

I believe this problem should be resolved, but I am leaving it here to see if actually it's not fixed and so people can report this if it pops up in their error log.